dev-overrides.yaml

```diff
@@ -12,8 +12,8 @@ modules:
                   ref: master
           - name: jboss-eap-7-image
             git:
-￼                   url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-￼                   ref: eap71-dev
+                  url: https://github.com/jboss-container-images/jboss-eap-7-image.git
+                  ref: eap71-dev
 
```

Comment:
A spacing change would be rejected in other cases, but as you're the maintainer, that's fine :P

Reply:
Thanks, will fix.

Reply:
Oh, actually this is correcting WS, so I'm ok with leaving it.